### PR TITLE
Sensor core module simplified

### PR DESCRIPTION
**Pull request by amaljofyzco**
sensor-core package after removing all sql injection references and code cleanup done.

### CODE REVIEW

File: sensor-core/build.gradle [hunk 1]
```diff
@@ -77,13 +77,16 @@ dependencies {
...
     implementation "com.google.android.material:material:1.3.0-alpha03"
     implementation "androidx.appcompat:appcompat:1.2.0"
 
     implementation 'com.github.PhilJay:MPAndroidChart:v3.1.0-alpha'
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
     implementation "com.google.android.gms:play-services-location:$play_services_location"
 
+    //activity recognition.
+    implementation 'com.google.android.gms:play-services-location:17.1.0'
+
```
Comment (avaidyam) [inline sensor-core/build.gradle]: Why does the sensor code have this many dependencies on UI libraries? Please remove as many dependencies as possible.

File: sensor-core/src/main/java/com/mindlamp/Accelerometer.java [hunk 2]
```diff
@@ -6,9 +6,6 @@
 import android.content.Context;
```
Comment (avaidyam) [inline sensor-core/src/main/java/com/mindlamp/Accelerometer.java]: There are many `.DS_Store` files that should not be checked into the repository, please add it to the `.gitignore` and remove them.

File: sensor-core/src/main/java/com/mindlamp/Accelerometer.java [hunk 3]
```diff
@@ -6,9 +6,6 @@
...
 import android.content.Intent;
```
Comment (avaidyam) [inline sensor-core/src/main/java/com/mindlamp/Accelerometer.java]: The package name should be `digital.lamp.____` and not `com.mindlamp.___`. Please rename/fix the references appropriately.

File: sensor-core/src/main/java/com/mindlamp/Gyroscope.java [hunk 4]
```diff
@@ -151,23 +137,23 @@ public void run() {
...
+//                        Intent newData = new Intent(ACTION_LAMP_GYROSCOPE);
+//                        sendBroadcast(newData);
+//                    }
+//                }).run();
+//            }
+//        } catch (SQLiteException e) {
+//            if (Lamp.DEBUG) Log.d(TAG, e.getMessage());
+//        } catch (SQLException e) {
+//            if (Lamp.DEBUG) Log.d(TAG, e.getMessage());
+//        }
```
Comment (avaidyam) [inline sensor-core/src/main/java/com/mindlamp/Gyroscope.java]: Can you remove all unused code?

File: sensor-core/src/main/java/com/mindlamp/Light.java [hunk 5]
```diff
@@ -124,23 +116,23 @@ public void onSensorChanged(SensorEvent event) {
...
+//                        Intent newData = new Intent(ACTION_LAMP_LIGHT);
+//                        sendBroadcast(newData);
+//                    }
+//                }).run();
+//            }
+//        } catch (SQLiteException e) {
+//            if (Lamp.DEBUG) Log.d(TAG, e.getMessage());
+//        } catch (SQLException e) {
+//            if (Lamp.DEBUG) Log.d(TAG, e.getMessage());
+//        }
```
Comment (avaidyam) [inline sensor-core/src/main/java/com/mindlamp/Light.java]: Please delete all unused code.

**Review by avaidyam — CHANGES_REQUESTED** [review]

File: sensor-core/build.gradle (hunk 1 above)
Comment (amaljofyzco) [reply sensor-core/build.gradle]: removed

File: sensor-core/src/main/java/com/mindlamp/Accelerometer.java (hunk 2 above)
Comment (amaljofyzco) [reply sensor-core/src/main/java/com/mindlamp/Accelerometer.java]: .gitignore file updated in sensor-core

File: sensor-core/src/main/java/com/mindlamp/Gyroscope.java (hunk 4 above)
Comment (amaljofyzco) [reply sensor-core/src/main/java/com/mindlamp/Gyroscope.java]: Done

File: sensor-core/src/main/java/com/mindlamp/Light.java (hunk 5 above)
Comment (amaljofyzco) [reply sensor-core/src/main/java/com/mindlamp/Light.java]: Done

File: sensor-core/src/main/java/com/mindlamp/Accelerometer.java (hunk 3 above)
Comment (amaljofyzco) [reply sensor-core/src/main/java/com/mindlamp/Accelerometer.java]: Done

File: sensor-core/src/main/java/com/mindlamp/Accelerometer.java (hunk 2 above)
Comment (amaljofyzco) [reply sensor-core/src/main/java/com/mindlamp/Accelerometer.java]: removed .DS_Store form git

**Pull request closed without merging**